docker/oso-clam-update/generate-containers.yml

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/ansible-playbook
+---
+- hosts: localhost
+  gather_facts: true
+
+  pre_tasks:
+  - name: Checking build environment
```

Comment:
checking build references rhel/fedora; but generate containers references rhel/centos.  Is fedora supported? tested?  Is centos supported? tested?

Reply:
We only use RHEL/Centos7, both are testing and working. generate-containers.yml is tools-managed and outside the scope of this PR.

Reply:
Why are there references to fedora if we don't use it?  Can it be removed?